docs/blog/2019-08-06-gazette-august/index.md

```diff
@@ -0,0 +1,125 @@
+
+Here's a convenient [playlist](https://www.youtube.com/playlist?list=PLz8Iz-Fnk_eTpvd49Sa77NiF8Uqq5Iykx) if you want to binge watch these.
+
+### Theme Jam
+
+Talking about jam, to celebrate the launch of Gatsby Themes, we announced the [Gatsby Theme Jam](https://themejam.gatsbyjs.org/) last month.
+
+The response was incredible. Seeing so many wonderful themes from all of you has been spectacular and we're afraid we might like _too_ many of them to pick one.
+
+The team is currently hard at work judging and picking their favourites so results should be out very soon!
+
+## Community Highlights
+
+Firstly, thank you for all your contributions, folks! Every single contribution to Gatsby (however small or big) makes it better. Some of the community highlights from the last month include:
+
+- [lannonbr](https://github.com/lannonbr) [built a Table of Contents component](https://github.com/gatsbyjs/gatsby/pull/15251) that is live on our documentation and is great when reading a lengthy documentation page
+- [kellyvaughn](https://github.com/kellyvaughn) [added support for metafields to product nodes](https://github.com/gatsbyjs/gatsby/pull/16312) in `gatsby-source-shopify`
+- [datakurre](https://github.com/datakurre) added a great [fix to ensure touchNode](https://github.com/gatsbyjs/gatsby/pull/15919) populates typeOwners
+- [grantglidewell](https://github.com/grantglidewell) added [support](https://github.com/gatsbyjs/gatsby/pull/14630) for Gatsby [Preview](https://www.gatsbyjs.com/preview) for Drupal in `gatsby-source-drupal`
+- [elken](https://github.com/elken) added [support for the Prism command-line](https://github.com/gatsbyjs/gatsby/pull/16170) plugin in `gatsby-remark-prismjs`
+- [cephalization](https://github.com/cephalization) added a [great error check](https://github.com/gatsbyjs/gatsby/pull/16272) in Gatsby
+
+Gatsby has always been a community effort. You make a _lot_ of contributions. In fact, we made a brand new record last month:
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">We&#39;ve merged in 500 PRs into <a href="https://twitter.com/gatsbyjs?ref_src=twsrc%5Etfw">@gatsbyjs</a> over the past month! 😱<br/><br/>Thank you for your contributions, folks! <a href="https://t.co/dsU85CzuGX">pic.twitter.com/dsU85CzuGX</a></p>&mdash; Sid (@chatsidhartha) <a href="https://twitter.com/chatsidhartha/status/1156334013247737856?ref_src=twsrc%5Etfw">July 30, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+## Gatsby Site of the month
+
+Time to show off some great stuff people have been building.
+
+The Gatsby site of the month is [www.fourpost.com](https://www.fourpost.com)
+
+We love it because:
+
+- It's so fast
```

Comment:
Let's quantify fast!

Here's a webpagetest with a Lighthouse run. We should _generally_ validate these things before we assert it's fast. It is fast, but it's not _perfect_ nor "Gatsby fast". Few too many tracking scripts :/

https://www.webpagetest.org/result/190807_6R_436af0f9843ced4a0ffb946dc7f47c54/

Reply:
You're absolutely right. Removed and replaced with what made me feel like it's fast in the first place (image loading)